cmd/imagedigestexporter/main.go

```diff
@@ -56,7 +61,13 @@ func main() {
 			// if this image doesn't have a builder that supports index.json file,
 			// then it will be skipped
 			logger.Infof("ImageResource %s doesn't have an index.json file: %s", imageResource.Name, err)
-			continue
+			logger.Info("Checking legacy image output path.")
+			legacyPath := filepath.Join(legacyOutputPath, imageResource.Name)
+			ii, err = layout.ImageIndexFromPath(legacyPath)
+			if err != nil {
+				continue
+			}
+			logger.Warn("index.json found at legacy path. Support for this location will be removed in a future release.")
```

Comment:
Could you include the legacy path as part of the error message to make it clear what the location is that's problematic? e.g. `fmt.Sprintf("index.json found at %q. Support for this legacy path will be removed in a future release.", legacyPath)`

Reply:
Done.